packages/reporters/dev-server/src/Server.js

```diff
@@ -495,6 +495,17 @@ export default class Server {
       setHeaders(res);
       next();
     });
+
+    app.use((req, res, next) => {
+      if (req.url === '/__parcel_healthcheck') {
```

Comment:
Is the `__` in `__parcel_healthcheck` meant to make it "private"?

Reply:
Yes, we're just trying to prevent clashes and be consistent with `__parcel_hmr`